### ISSUE TITLE
Move Connect button to top of mobile settings UI

### ISSUE THREAD

**Issue by aj47** [aj47/SpeakMCP#982]
The Connect button is currently at the bottom of the settings screen on mobile. It should be moved to the top for better accessibility and discoverability.

Currently users have to scroll down to find the Connect button in settings, but this is one of the most important actions so it should be prominently placed at the top.
